benchbuild/project.py

```diff
@@ -161,6 +165,7 @@ class Project(metaclass=ProjectDecorator):
     VERSION = None
     SRC_FILE = None
     CONTAINER = None
+    SLURM_OPTIONS: List[SlurmOption] = []
```

Comment:
Oh, this could make things a bit more difficult. A lot of the options have specific slurm semantics like `Hint`. And also for Experiments where we need to specify slurm requirements, where the expr needs multi node/sockets, just cores is not the same as cores per socket or node. I currently see no easy way to generalize this.

Reply:
Easy. I would not change anything for this PR. Later we can add an adapter (lllllaater) that just filters out hints that have no meaning for the given platform (slurm/cgroups)